setup/src/Magento/Setup/Module/I18n/Parser/Adapter/Html.php

```diff
@@ -16,8 +16,9 @@ class Html extends AbstractAdapter
      * Covers
      * <span><!-- ko i18n: 'Next'--><!-- /ko --></span>
      * <th class="col col-method" data-bind="i18n: 'Select Method'"></th>
+     * <translate args="'Items in Cart'"/>
      */
-    const HTML_FILTER = "/i18n:\s?'(?<value>[^'\\\\]*(?:\\\\.[^'\\\\]*)*)'/i";
+    const HTML_FILTER = "/i18n:\s?'(?<value_i18n>[^'\\\\]*(?:\\\\.[^'\\\\]*)*)|translate args=\"'(?<value_translate_args>[^\"\\\\]*(?:\\\\.[^\"\\\\]*)*)'\"/i";
```

Comment:
It would be better deprecate this constant and add the new array with regular expressions for different cases with translation match. This approach will make the translations collecting process more clear.